cfn-resources/auditing/cmd/resource/resource.go

```diff
@@ -202,20 +205,14 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		}, nil
 	}
 
-	atlasAuditing, res, err := client.Auditing.Configure(context.Background(), *currentModel.ProjectId, &auditingInput)
-
-	if err != nil {
-		_, _ = log.Debugf("Create - error: %+v", err)
-		return progress_events.GetFailedEventByResponse(err.Error(), res.Response), nil
-	}
+	atlasAuditing, res, err := atlasV2.AuditingApi.UpdateAuditingConfiguration(context.Background(), *currentModel.ProjectId, &auditingInput).Execute()
 
 	if err != nil {
 		_, _ = log.Debugf("Update - error: %+v", err)
```

Comment:
if we don't want the result, we can just do:   log.Debugf("Update - error: %+v", err)

Reply:
for reference it is associated to a linting check: https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/711#discussion_r1332653450

Reply:
yes, discussed in the other PR, it's ok as it is now, thanks